tensorflow/java/src/main/native/tensor_jni.h

```diff
@@ -24,11 +24,19 @@ extern "C" {
 
 /*
  * Class:     org_tensorflow_Tensor
+ * Method:    buffer
+ * Signature: (J)Ljava/nio/ByteBuffer;
+ */
+JNIEXPORT jobject JNICALL Java_org_tensorflow_Tensor_buffer(JNIEnv *, jclass,
```

Comment:
Nit: Place this method in the `.cc` and `.h` files after `allocate` and `delete`